nautobot/docs/installation/nautobot.md

```diff
@@ -0,0 +1,255 @@
+From here on out, anytime you become `nautobot`, your `NAUTOBOT_ROOT` will be set automatically.
+
+## Activate the Virtual Environment
+
+!!! warning
+    This and all remaining steps in this document should all be performed as the `nautobot` user!
+
+    Hint: Use `sudo su - nautobot`
+
+To work inside a Python virtualenv, it must be activated. This makes sure that the version of Python you're using, as well any dependencies that you install remain isolated in this environment. 
+
+If it helps, try to think of activating the virtualenv like entering a shell unique to Nautobot's application environment.
+
+```no-highlight
+$ source /opt/nautobot/bin/activate
+(nautobot) $
+```
+
+Observe that after activating, your prompt will now be preceded with the name of the virtualenv (`nautobot`).
+
+!!! note
+    From here on out, any time you see a command prompt preceded by `(nautobot)`, that is your indicator that you should be activated in the virtualenv.
+
+## Prepare the Virtual Environment
+
+Before we install anything into the virtualenv, we want to make sure that Pip is running the latest version. This is neecessary because sometimes when a new virtualenv is created, a cached version of Pip is installed into it. (Yes, even after we've deliberately upgraded Pip at the system level!)
+
+We also want to deliberately install the `wheel` library which will tell Pip to always try to install wheel packages if they are available. A [wheel is a pre-compiled Python package](https://realpython.com/python-wheels/), which is quicker and safer to install because it does not require development libraries or `gcc` to be installed on your system just so that some more advanced Python libraries can be compiled. 
+
+```no-highlight
+(nautobot) $ pip3 install --upgrade pip wheel
+```
+
+## Install Nautobot
+
+Use Pip to install Nautobot:
+
+```no-highlight
+(nautobot) $ pip3 install nautobot
+```
+
+Great! We have a virtualenv ready for use by the `nautobot` user, so let's proceed to verifying the installation.
+
+## Verify your Nautobot Installation
+
+You should now have a fancy `nautobot-server` command in your environment. This will be your gateway to all things Nautobot! Run it to confirm the installed version of `nautobot`:
+
+```no-highlight
+(nautobot) $ nautobot-server --version
+```
+
+## Configuration
+
+Before you can use Nautobot, you'll need to configure it by telling it where your database and Redis servers can be found, among other things. This is done with the `nautobot_config.py` configuration file.
+
+### Initialize your configuration
+
+Initialize a new configuration by running `nautobot-server init`. You may specify an alternate location and detailed instructions for this are covered in the documentation on [Nautobot Configuration](../../configuration).
+
+However, because we've set the `NAUTOBOT_ROOT`, this command will automatically create a new `nautobot_config.py` at the default location based on this at `/opt/nautobot/nautobot_config.py`:
```

Comment:
This created the config at `/opt/nautobot/.nautobot/nautobot_config.py` not `/opt/nautobot/nautobot_config.py`

Reply:
That's because you didn't install the patch from this PR. :)

Reply:
Back where it says `pip3 install nautobot` do `pip3 install git+https://github.com/jathanism/nautobot@jathanism-quickstart` instead.

Reply:
ack no worries